controllers/users.js

```diff
@@ -64,7 +64,9 @@ const getUserById = async (req, res) => {
 const getUsers = async (req, res) => {
   try {
     const allUsers = await userQuery.fetchUsers(req.query);
-
+    if (!allUsers.length) {
+      return res.boom.notFound("User doesn't exist");
+    }
     return res.json({
       message: "Users returned successfully!",
       users: allUsers,
```

Comment:
I think here we should return an empty array if no users are found for the request with a status code of 200 ok, instead of returning 404, we are returning multiple users, so it should be like [ ] or [user1] or [user1, user2...]

Reply:
removed it